WRITING.md

```diff
@@ -154,6 +155,67 @@ This will render **Astro's latest version: v1.2.1**.
 
 The `<Version />` component is currently used in our Integrations pages as a way to keep each integration’s version up-to-date without having to merge changes to these pages directly. It's worth noting that this component is only updated at build-time, thus a package's version will not change until the site is rebuilt, be it manually or because a new PR was merged into main.
 
+### Tabs Components
+
+For pieces of content relevant to only a percentage of users (e.g. specific package manager or UI framework guidance), we have Tabs components to help you.
+
+For the two common cases: package managers and UI frameworks, the corresponding `<PackageManagerTabs>` and `<UIFrameworkTabs>` components already got you covered.
+
+It's worth mentioning that these components share state, so if you change the active tab of one `<PackageManagerTabs>` component, all of the other package manager Tabs components on the same page will also change.
+
+#### Examples
+
+All you have to do to use a Tabs component is to import it into the file and fill the available slots with content, for example:
+
+````jsx
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # create a new project with npm
+  npm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # create a new project with pnpm
+  pnpm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # create a new project with yarn
+  yarn create astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+````
+You can take a look at how this example will be rendered on the [Automatic Astro Installation](https://docs.astro.build/en/install/auto/#1-run-the-setup-wizard) page.
+
+If necessary, you can also create your own Tabs component using the base `Tabs.tsx` component. To do this, create a new Astro component in the [`src/components/tabs`](https://github.com/withastro/docs/blob/main/src/components/tabs/) directory and fill `Tabs.tsx` with the corresponding tabs and panel slots, like the example below:
```

Comment:
``````suggestion
If necessary, you can also create your own custom Tabs component using the base `Tabs.tsx` component. To do this, create a new Astro component in the [`src/components/tabs`](https://github.com/withastro/docs/blob/main/src/components/tabs/) directory, e.g. `MyCustomTabs.jsx`. (Do not use `<Tabs>` directly in a Markdown page. Create your own component instead.)

Inside `MyCustomTabs.jsx`, import the Tabs component and create one `<Tabs>` component. Be sure to include the `client:visible` directive and give a unique name to the `sharedStore`. Each created Tabs component should have its own `sharedStore` to avoid unrelated tabs changing one another accidentally.

```astro
---
import Tabs from './Tabs';
---
<Tabs client:visible sharedStore="my-shared-store">
</Tabs>
```
To create your custom tab component, follow the pattern below using a `<Fragment>` with a named slot for each tab and panel to be created. Note that you must define your tab names here (e.g. Preact, React, Solid, Svelte, Vue), but the content for each panel will be written when your custom component is imported and used in a Markdown page, as in the previous `<PackageManagerTabs>` example.
``````